openff/toolkit/tests/test_molecule.py

```diff
@@ -3615,6 +3619,110 @@ def test_apply_elf_conformer_selection(self, toolkit):
             initial_conformers[1].value_in_unit(unit.angstrom),
         )
 
+    def test_make_carboxylic_acids_cis(self):
+        # First, check that we get exactly the right coords for cis and trans formic acid
+        offmol = Molecule.from_mapped_smiles(
+            "[H:5][C:1]([H:6])([H:7])[C:2](=[O:4])[O:3][H:8]"
```

Comment:
```suggestion
            "[H:1][C:2](=[O:3])[O:4][H:5]"
```

Maybe swap to formic acid for simplicity?